modules/osdk-init-resource.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-init-resource_{context}"]
+= Initializing required custom resources
+
+An Operator might require the user to instantiate a custom resource before the
```

Comment:
s/the user/you

Reply:
Audience here is Operator developer/authors.